lib/restAPIs/library.js

```diff
@@ -0,0 +1,105 @@
+//const assert = require('assert');
+
+//const assertParam = require('../util/assertParam');
+const RestUtil = require('../util/RestUtil');
+const buildCommonQuery = require('../util/buildCommonQueryParams');
+
+module.exports = class RestTopic extends RestUtil {
+  /**
+   * Get base endpoint for this API category
+   */
+  _getBase() {
+    return 'library';
+  }
+
+  /**
+   * Return object representing the library for the authenticated user.
+   * @param {array[String]} outputFlag - Filtered output. DEAFULT = include everything, FILTER_TOC = filter out cahpters and pages.
+   * @param {String} fields
+   */
+  getLibrary(outputFlag = 'DEFAULT', fields = '') {
+    const queryParameter = buildCommonQuery(fields);
+    queryParameter.outputFlag = outputFlag;
+
+    return this._makeRequest(
+      `${this._getBase()}`,
+      queryParameter,
+      'GET',
+      null,
+      null
```

Comment:
If you're not setting the last 2 parameters, you can simply exclude them in your makeRequest() call.

Reply:
fixed.